docs/getting-started/features/tamper-detection.en.md

```diff
@@ -9,76 +9,62 @@ To secure your Krux device, always verify firmware authenticity before installat
+## Tamper Check Flash Hash (TC Flash Hash) - A Tamper Detection Tool
 
-The tool generates a unique image and four tamper detection words based on a hash of your *TC Code*, the device's UID, and the flash content. The flash memory is divided into two regions:
+The *TC Flash Hash* tool enables you to verify if the device's internal flash memory content has been altered. This tool generates a unique image and two sets of two tamper detection words based on a hash of your *TC Code*, the device's UID, and its internal flash content. The flash memory is divided into two regions:
 
-- **Firmware Region:** Generates the image and the first two words.
+- **Firmware Region:** The area only filled with firmware code. It generates the memorable image and the first set of two words.
 
-- **User's Region:** Generates the last two words.
+- **User's Region:** The area used to stored encrypted mnemonics, settings and TC Code. It generates the last set of two words.
 
 <div style="text-align: center;">
     <img src="../../../img/flash_hash.bmp" alt="TC Flash Hash" width="200"/>
 </div>
 
-*Example: The blue symbol and words 'tail monkey' represent the firmware region, while 'wrestle over' reflects the user region.*
+*Example: The blue symbol and words 'tail monkey' represents the firmware region, while 'wrestle over' user's region.*
 
-Any change in the flash content results in a different image or words:
+Any change in the flash content results in a different image and words:
 
-- **Firmware Changes:** Alterations in the firmware region, including the bootloader, change the image and the first two words.
+- **Firmware Changes:** Alterations in the firmware region, including the bootloader, change the image and the first set of two words.
 
-- **User's Data Changes:** Modifications in the user's region, such as new settings or stored mnemonics, change the last two words.
+- **User's Data Changes:** Changes in the user's region, such as new settings or stored mnemonics, change the last set of two words.
 
-- ***TC Code* Changes:** Replacing the *TC Code* alters the image and all four words.
+- ***TC Code* Changes:** Replacing the *TC Code* alters the image and all sets of words.
 
-### Filling Empty Flash Blocks
+### Filling Empty Flash Memory Blocks
 
-Krux performs a memory sweep while simultaneously capturing a live feed from the camera. Whenever an empty block is found in the flash memory, Krux estimates the image's entropy by evaluating its color variance. Krux waits until minimum threshold is met, then uses the data from the image to fill these empty spaces with rich, random entropy.
+Use this to enhance tamper detection. Krux performs a memory sweep while capturing a live feed from the camera. Whenever an empty block is found in the flash memory, it uses the data from the image to fill these empty spaces with rich, random entropy. It estimates the image's entropy by evaluating its color variance waiting until a minimum threshold is met. 
```

Comment:
IMO original version text flow better reflect sequence of events. But second is OK

Reply:
I think we are used to speaking programmatically, I believe that hiding this information to reveal it later makes it easier for the average user to understand. Even if this is not the correct order of events, we are detailing what was done right after.